src/Group.php

```diff
@@ -0,0 +1,30 @@
+<?php
+namespace Yiisoft\Router;
+
+class Group
```

Comment:
How about test for Group?

Reply:
Well, I think this one is unlikely to be merged. It's better to think on how to support native groups that are implemented in many routers.

Reply:
It can be implemented in adapter.

```php
class FastRouteAdapter implements \Yiisoft\Router\RouterInterface
{
    public function addGroup(\Yiisoft\Router\Group $group) 
    {
        $this->fastRoute->addGroup($group->getPrefix(), function (RouteCollector $r) use ($group) {
            foreach ($group->getRoutes() as $route) {
                $r->addRoute($route->getMethod(), $route->getPattern(), $route->getMiddleware());
            }
        });
    }
}
```

Reply:
Re-checked. You're right.

Reply:
There was a test for group class.